src/containers/Statistics/StatisticsCard.tsx

```diff
@@ -0,0 +1,35 @@
+import { StatisticCardAll } from '@components/Statistics/StatisticCardAll'
+import React from 'react'
+import { useSelector } from 'react-redux'
+import solanaConnectionSelectors from '@selectors/solanaConnection'
+interface Props {
+  collateral: number
+  valume: number
+  mint: number
+  debt: number
+  fee: number
+}
+
+export const StatisticsCard: React.FC = () => {
+  const network = useSelector(solanaConnectionSelectors.network)
+  const [data, setData] = React.useState<Props>({
+    collateral: 0,
+    valume: 0,
+    mint: 0,
+    debt: 0,
+    fee: 0
+  })
+
+  const updateData = async () => {
+    await fetch(`https://api.synthetify.io/stats/${network.toLowerCase()}`)
+      .then(async response => await response.json())
```

Comment:
is async function really needed there? i think you can merge this two then calls into one